distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1029,7 +1093,8 @@ type Integer
     bit_shift : Integer -> Integer ! Arithmetic_Error
     bit_shift self that = @Builtin_Method "Integer.bit_shift"
 
-    ## Performs a left-wise bit shift on the bits of this.
+    ## GROUP Bitwise
+       Performs a left-wise bit shift on the bits of this.
```

Comment:
Again, not really this PR but the doc changes highlight these.

I was not familiar with the phrase 'left-wise bit shift'. Trying to Google it I find some results but not much. Would a 'bit-wise left shift' be possibly more familiar to users?